spec/schema-embed_content_request.json

```diff
@@ -70,6 +70,11 @@
         "preset"
       ]
     },
+    "content_id": {
+      "type": "string",
+      "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$",
```

Comment:
I think this should be no different than using `format: uuid`?

Reply:
On another note, this presents some inconsistencies in the schema, because we previously did not validate `id` as a UUID, when the intent that it is. I think we should be consistent overall

Reply:
💯 
> On another note, this presents some inconsistencies in the schema, because we previously did not validate `id` as a UUID, when the intent that it is. I think we should be consistent overall

@bjester I think I did try `format: uuid` but this didn't seem to work as expected on running the tests thus the above. It's very likely that the uuid format is not present in the  current version of json-schema we are using?

Reply:
Ah you're right. Draft-07 doesn't have it and I think we're limited by `jsonschema` which at version 3.2.0 doesn't have it but that's the last version that supports python 3.6. If we think about the future, using a regex like you've done can easily be replaced by `format: uuid` once we are able to upgrade. So perhaps we create a new definition for it so we can reuse the regex for `content_id` and `id`? And soon for `channel_id` for which I have an issue assigned to add that.

Reply:
I agree 💯